adding the new feature to report fraud for mfa

lem0w opened the pull request:
Required items, please complete
   
   Change(s):
   - added has fraud for MFA and reference to the MS docs

   Reason for Change(s):
   - user can report fraud on several ways, app prompt, phone code enter, this is mainly to increase the visibility of a deny from the user when prompted for MFA.

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - haven't been able to make it run.
---------------------------------------------------------------------------------------------------------

devikamehra commented:
The query has been migrated to another folder and thus any changes required should be made to the rule inside the solution folder https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Active%20Directory/Analytic%20Rules/MFARejectedbyUser.yaml.

lem0w commented:
> The query has been migrated to another folder and thus any changes required should be made to the rule inside the solution folder https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Active%20Directory/Analytic%20Rules/MFARejectedbyUser.yaml.

Thanks, I have updated the correct file and reverted the previous changes
